ts/components/services/ServiceMetadata.tsx

```diff
@@ -0,0 +1,165 @@
+import TouchableDefaultOpacity from "../TouchableDefaultOpacity";
+import { capitalize } from "../../utils/strings";
+import customVariables from "../../theme/variables";
+import { ItemAction } from "../../utils/url";
+
+import ItemSeparatorComponent from "../ItemSeparatorComponent";
+
+import SectionHeader from "./SectionHeader";
+import LinkRow from "./LinkRow";
+
+type Props = {
+  address?: string;
+  androidStoreUrl?: string;
+  email?: string;
+  fiscalCode?: string;
+  getItemOnPress: (value: string, valueType?: ItemAction) => () => void;
+  iosStoreUrl?: string;
+  pec?: string;
+  phone?: string;
+  serviceId: string;
+  supportUrl?: string;
+  webUrl?: string;
+};
+
+const styles = StyleSheet.create({
+  valueHeader: {
+    marginTop: customVariables.spacerLargeHeight
+  },
+  valueRow: {
+    flexDirection: "column"
+  },
+  valueItem: {
+    flexDirection: "row",
+    flex: 1,
+    justifyContent: "space-between",
+    marginVertical: 24
+  },
+  itemLabel: {
+    flex: 1
+  },
+  itemValue: {
+    flex: 1,
+    marginRight: 8
+  }
+});
+
+/**
+ * Renders a dedicated section with a service's metadata and the header.
+ */
+const ServiceMetadata: React.FC<Props> = ({
+  address,
+  androidStoreUrl,
+  email,
+  fiscalCode,
+  getItemOnPress,
+  iosStoreUrl,
+  pec,
+  phone,
+  serviceId,
+  supportUrl,
+  webUrl
+}: Props) => (
+  <>
+    <SectionHeader iconName="io-phone" title={"services.contactsAndInfo"} />
+
+    {/* links */}
+    {webUrl && <LinkRow text={"services.visitWebsite"} href={webUrl} />}
+    {supportUrl && (
+      <LinkRow text={"services.askForAssistance"} href={supportUrl} />
+    )}
+    {iosStoreUrl && (
+      <LinkRow text={"services.otherAppIos"} href={iosStoreUrl} />
+    )}
+    {androidStoreUrl && (
+      <LinkRow text={"services.otherAppAndroid"} href={androidStoreUrl} />
+    )}
+
+    {/* touchable rows */}
+    {fiscalCode &&
+      renderInformationRow(
+        fiscalCode,
+        "profile.fiscalCode.fiscalCode",
+        getItemOnPress(fiscalCode, "COPY"),
+        "clipboard.copyText"
+      )}
+    {address &&
+      renderInformationRow(
+        address,
+        "services.contactAddress",
+        getItemOnPress(address, "MAP"),
+        "openMaps.openAddressOnMap"
+      )}
+    {phone &&
+      renderInformationRow(
+        phone,
+        "global.media.phone",
+        getItemOnPress(`tel:${phone}`),
+        "messageDetails.call"
+      )}
+    {email &&
+      renderInformationRow(
+        email,
+        "global.media.email",
+        getItemOnPress(`mailto:${email}`),
+        "messageDetails.sendEmail"
+      )}
+    {pec &&
+      renderInformationRow(
+        pec,
+        "global.media.pec",
+        getItemOnPress(`mailto:${pec}`),
+        "messageDetails.sendEmail"
+      )}
+    {renderInformationRow(
+      serviceId,
+      "global.id",
+      getItemOnPress(serviceId, "COPY"),
+      "clipboard.copyText"
+    )}
+  </>
+);
+
+const renderInformationRow = (
```

Comment:
Should the content be align to right?

| this PR  | figma |
| ------------- | ------------- |
![this PR](https://user-images.githubusercontent.com/822471/125652712-775bb28b-4d56-4898-a157-d01217a7785b.png) | ![figma](https://user-images.githubusercontent.com/822471/125652802-dea8b10f-4319-4b41-b230-1ca5f631dde5.png)

Reply:
Wow, this is extremely embarassing 🔴  I can only say **thank you!!!**.
Here's what looks like now:

![Screenshot 2021-07-15 at 15 08 55](https://user-images.githubusercontent.com/2094604/125796937-59d9702c-e0c8-4494-a35e-a8a49de13d9c.png)